.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["esbenp.prettier-vscode", "dbaeumer.vscode-eslint"]
```

Comment:
List of extensions that are recommended to be used for VSCode together with our Prettier and ESLint setup. Learn more about the `extensions.json` file here: https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions